kmod/core/core.c

```diff
@@ -1008,9 +1008,22 @@ int kpatch_register(struct kpatch_module *kpmod, bool replace)
 		func->op = KPATCH_OP_NONE;
 	} while_for_each_linked_func();
 
+/* HAS_KERNEL_TAINT - upstream 2992ef29ae01 "livepatch/module: make TAINT_LIVEPATCH module-specific" */
+#ifdef RHEL_RELEASE_CODE
+# if RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(7, 4)
+#  define HAS_KERNEL_TAINT
```

Comment:
I couldn't think of a really good name, so I thought ```HAS_KERNEL_TAINT``` would imply that the kernel was the actor applying the taint flag, not the module itself.  If ```HAS_MODULE_TAINT``` is easier to remember (ie, module _specific_ taint), then I can certainly change it.